src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -37,6 +38,14 @@ public static class Keywords
         private void RequestStart(string scheme, string host, int port, string pathAndQuery, byte versionMajor, byte versionMinor, HttpVersionPolicy versionPolicy)
         {
             Interlocked.Increment(ref _startedRequests);
+            if (!GlobalHttpSettings.DiagnosticsHandler.DisableUriRedaction)
+            {
+                int queryIndex = pathAndQuery.IndexOf('?');
+                if (queryIndex >= 0)
```

Comment:
This is different from what ILogger is doing ATM:

https://github.com/dotnet/runtime/blob/0f2e63fbdd63bc239df46efd01a329b42fea8531/src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs#L173-L174

I guess we need to align in one way or another?

Reply:
Fixed.